integrationExamples/gpt/gpt_sizemapping.html

```diff
@@ -0,0 +1,251 @@
+                            pageURL : 'http://drudgereport.com',
+                            refererURL : 'http://drudgereport.com',
+                            jstag_url : 'http://ox-d.intermarkets.net/w/1.0/jstag'
+                        }
+                    },
+                    {
+                        bidder: 'pubmatic',
+                        params: {
+                            publisherId: 39741,
+                            adSlot: '39620189@300x250'
+                        }
+                      
+                    },
+                     {
+                        bidder: 'criteo',
+                        params: {
+                            nid: '2612',
+                            cookiename: 'cto_topix',
+                            varname : 'crtg_content'
+                        }
+                      
+                    },
+                  
+                    {
+                        bidder: 'yieldbot',
+                        params: {
+                            pub: 'id',
+                            name: 'name'
+                        }
+                    }, 
+                    {
+                        bidder: 'amazon',
+                        params: {
+                            aId : 3080
+                        }
+                    }      
+                ]
+        }
+
+        rightSlotSizes = pbjs.getSizeByDynamicSizes(dynamicSizes);
+
+        var adUnit2 = {
+            code: rightSlotCode,
+            sizes: rightSlotSizes,
+            bids: [{
+                        bidder: 'appnexus',
+                        params: {
+                            placementId : '5215561'
+                        }
+                    },
+                    {
+                        bidder: 'appnexus',
+                        params: {
+                            placementId: '2251610',
+                            referrer:'rolando.com',
+                            alt_referrer:'rolando.com'
+                        }
+                    }]
+
+            };
+
+        var adUnit3 = {
+            code: topSlotCode,
+            bids: [{
+                        bidder: 'appnexus',
+                        params: {
+                            placementId : '5215561'
+                        }
+                    },
+                    {
+                        bidder: 'appnexus',
+                        params: {
+                            placementId: '2251610',
+                            referrer:'rolando.com',
+                            alt_referrer:'rolando.com'
+                        }
+                    }]
+
+            };
+
+
+        //add array of ad units
+         var arr = [adUnit2, adUnit1];
+         pbjs.addAdUnits([adUnit2, adUnit1]);
+
+         var ad2 = pbjs.addAdUnits(adUnit3);
+         //add size mapping
+         ad2.addSizeMapping([540, 480], [120, 60]);
+         ad2.addSizeMapping([111, 111], [30, 60]);
```

Comment:
case3) add dynamic sizes using addSizeMapping()

Case1 and 3 will set sizes when the requestBid is called.